circuit-lib/circuitlib-rs/circuits/merkle/MerkleTreeProof.circom

```diff
@@ -0,0 +1,49 @@
+pragma circom 2.1.4;
+
+include "../../node_modules/circomlib/circuits/poseidon.circom";
+include "../../node_modules/circomlib/circuits/comparators.circom";
+include "../../node_modules/circomlib/circuits/bitify.circom";
+include "../../node_modules/circomlib/circuits/switcher.circom";
```

Comment:
I think it would be cleaner to put the circuits into circuit-lib.circom, then we don't need to have the build script twice as well.
On the imports, you can use the -l option of the circom compiler to link folders to import from (see circuit-lib.circom).
`` circom example.circom -l node_modules/circom-lib/circuits/poseidon``